appveyor.yml

```diff
@@ -118,11 +118,11 @@ deploy:
     on: 
       branch: master
   - provider: GitHub
-    tag: Rubberduck v$(appveyor_build_version)
+    tag: Prerelease-v$(appveyor_build_version)
     release: $(release_name)
     draft: false
     prerelease: true
-    description: "AppVeyor build on [$(appveyor_repo_branch)] - https://ci.appveyor.com/project/rubberduck-vba/rubberduck/build/$(appveyor_build_version)<br/><br/>Merges #$(appveyor_pull_request_number) - $(appveyor_pull_request_title)"
+    description: "AppVeyor build on [$(appveyor_repo_branch)] - https://ci.appveyor.com/project/rubberduck-vba/rubberduck/build/$(appveyor_build_version)"
```

Comment:
Bummer this isn't working. Would have been useful to see what's what just browsing the releases.

Reply:
Ya.that was the idea